engine/component.ftl

```diff
@@ -137,6 +137,25 @@
     }
 ] ]
 
+[#macro addResourceGroupInformationExtensions type extensions provider]
```

Comment:
Not sure the name quite portrays what is happening here. Maybe `addResourceGroupAttributeValues` or something like that. At the moment we are introducing a new concept, extensions which we haven't used anywehere else. Where attributes are know and values is what we are changing

Reply:
Yep agreed, I kept changing this around and figgured it would be better to collaborate on a name. I've implemented your recommendation in cb51fca